08_project/mohankrishna/cmd/mohankrishna-puppy-server/main.go

```diff
@@ -0,0 +1,54 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+
+	types "github.com/anz-bank/go-training/08_project/mohankrishna/pkg/mohankrishna-puppy"
```

Comment:
Better to have the package at `github.com/anz-bank/go-training/08_project/mohankrishna/pkg/puppy` and import it with the default name `puppy`.